List page design integration

SatvikS7 opened the pull request:
Overall, formats the new list page, including filtering and sorting, in line with the figma designs. Notable changes include:
- animation drop-down when the filter button is clicked
- new header section for the filter options
- alignment of entry information under the proper columns
- slight button animation when hovered

list-page can move into the main branch; however, the following should be addressed soon:
- stylization of the filter header as per the figma files
- stylization of the modal pop-up
- tag search functionality with multiple tags
- #42

y7chiu commented:
I think we should get rid of "tags:" on the list entry and move tags, timestamp, and Delete to the center of category name.

y7chiu reviewed (changes requested):
I think we should get rid of "tags:" on the list entry and move tags, timestamp, and Delete to the center of category name. And also for the drop-down filter, are we going to move it to the left that aligns the search bar?
And, the placehold of the search bar should be changed.
Otherwise, I think it's good!

SatvikS7 commented:
> I think we should get rid of "tags:" on the list entry and move tags, timestamp, and Delete to the center of category name. And also for the drop-down filter, are we going to move it to the left that aligns the search bar? And, the placehold of the search bar should be changed. Otherwise, I think it's good!

Fixed "tags:", timestamp, delete button, title alignment, changed the search bar place holder. We can change the drop-down filter but that can be done with future iterations.
